.devcontainer/scripts/container-creation.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+# GitHub doesn't pull in submodules by default when mounting
+# the repo directory on the container so we do this post-create
+git submodule update --init --recursive
+# Run restore to install the SDK before the app starts
+./restore.sh
```

Comment:
@dougbu Is there a way for us to use the `./restore.sh` script exclusively for the SDK installation and not actually restore anything? That would also trim down the restart time?

I know I could probably do the fetching of the dotnet-install script and invocation here but I want to avoid reusing code.

Reply:
I'm not sure of the scenario here but think you want to install just the .NET SDK as part of container creation❔

If that's the case, the following command is about as minimal as you can get:

``` bash
./eng/build.sh --only-build-repo-tasks
```

This will install the SDK and runtimes specified in global.json, install `dotnet-serve` and `playwright-sharp-tool`, restore and build the RepoTasks project, and recreate the artifacts/bin/GenerateFiles/Directory.Build.* files. Basically, it'll ensure the repo is ready for real work.

Reply:
Yep! This is exactly what I was looking for.